test/regress/cli/regress0/ff/proj-issue704.smt2

```diff
@@ -0,0 +1,8 @@
+; SCRUBBER: grep -w 'sat'
```

Comment:
Is this line actually required if the status is set? The regression script should be able to check the output.

Reply:
This line trims the output of `get-value`. I expect that the exact form of the output is unstable, so I didn't want to check for it.